patient_portal/get_appointments.php

```diff
@@ -54,8 +54,9 @@
         $dispmin  = substr($row['pc_startTime'], 3, 2);
         if ($disphour >= 12) {
             $dispampm = "pm";
```

Comment:
AM/PM is not a standard in Medical settings, or at all for most countries:  Just former British colonies.

> 
> The 24-hour clock is the convention of time keeping in which the day runs from midnight to midnight and is divided into 24 hours, indicated by the hours passed since midnight, from 0 to 23. This system is the most commonly used time notation in the world today,[1] and is used by international standard ISO 8601.[2]
> 
> A limited number of countries, particularly English-speaking nations, use the 12-hour clock as a standard, or a mixture of the 24- and 12-hour time systems. In countries where the 12-hour clock is still dominant, some professions prefer to use the 24-hour clock. For example, in the practice of medicine the 24-hour clock is generally used in documentation of care as it prevents any ambiguity as to when events occurred in a patient's medical history.[3] In the United States and a handful of other countries, it is popularly referred to as military time.[4][1]

Reply:
This is old code that was reused. The add_edit_event has the same items